CONTRIBUTING.md

```diff
@@ -109,6 +113,13 @@ Thank you! We'll try to respond as quickly as possible.
    by executing `bin/test`. (Initially, this will take a while to compile
    everything.)
 
+   - One thing to note is that this script optionally runs Clippy checks as well.
+   Since this requires the nightly compiler these checks are disabled by default.
+   In order to run these checks you can do:
+   ```bash
+   bin/test -- --with-clippy
```

Comment:
Change this to something like `--disable-clippy` as Sean mentioned in the gitter channel.